src/index.html

```diff
@@ -1,13 +1,64 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<!DOCTYPE html>
+<html lang="en" class="html">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content=" width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+    <title>Product cards</title>
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Imac image"
+        class="card__image"
+      >
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating" >
+        <div
+        class="card__stars stars stars--4"
+        >
```

Comment:
```suggestion
        <div class="card__stars stars stars--4">
```

Reply:
You can place it on one line, it's only one attribute here and that line is looks shorter than 80 symbols

Reply:
I also wanted one line, but lint gave me an error if I wrote in one line